app/resources/v1/reception_resource.rb

```diff
@@ -20,18 +20,31 @@ def request_attributes=(request_parameters)
 
       @model.request_attributes = request_parameters.map do |request|
         request.permit(request: permitted_request_attributes,
-                       sample: %i[name external_id species], container: %i[type barcode position])
+                       sample: %i[name external_id species study_uuid priority_level],
+                       container: %i[type barcode position])
                .to_h
                .with_indifferent_access
       end
     end
 
     def construct_resources!
       @model.construct_resources!
+      publish_messages
     end
 
     def permitted_request_attributes
       [*::Pacbio.request_attributes, *::Ont.request_attributes, *::Saphyr.request_attributes].uniq
     end
+
+    def publish_messages
+      Messages.publish(
+        @model.requests.map(&:sample),
+        Pipelines.reception.sample.message
+      )
+      Messages.publish(
+        @model.requests,
+        Pipelines.reception.stock_resource.message
+      )
+    end
```

Comment:
So two messages have to be sent to the MLWH? One for the sample table and one for stock resource? Just wondering whether a comment can be added to explain this, and maybe refactoring this a little e.g. `def publish_message(message, config)` and call that twice with the different params